src/Sentry.NLog/SentryTarget.cs

```diff
@@ -259,12 +259,11 @@ protected override void Write(LogEventInfo logEvent)
             }
 
             var exception = logEvent.Exception;
-            var formatted = Layout.Render(logEvent);
+            var formatted = RenderLogEvent(Layout, logEvent);
             var template = logEvent.Message;
 
-            var contextProps = GetAllProperties(logEvent);
-
             var shouldOnlyLogExceptions = exception == null && IgnoreEventsWithNoException;
+            var shouldIncludeProperties = ContextProperties?.Count > 0 || ShouldIncludeProperties(logEvent);
```

Comment:
is `IncludeEventProperties` irrelevant now? If so, please remove it.
Do we need to override `ShouldIncludeProperties` here?
We're preparing for 2.0.0 so breaking changes are welcome.

Reply:
`IncludeEventProperties` lives on `TargetWithContext`. Cannot be removed.  `TargetWithContext.ShouldIncludeProperties()` checks `IncludeEventProperties`.

Reply:
Got it. Thanks for clarifying